src/context/CapturesContext.js

```diff
@@ -131,6 +132,42 @@ export function CapturesProvider(props) {
     setCaptures(response.data);
   };
 
+  const getAllCaptures = async (filterInfo = {}) => {
+    log.debug('load all captures');
+    console.log('captures filterInfo -- ', filterInfo);
+
+    // if filterInfo contains new values override the defaults in state hooks
+    const {
+      filter = new FilterModel(),
+      orderBy = 'id',
```

Comment:
We should use the same order as the table: `order: [`${orderBy} ${order}`]`
This is particularly important when the user hits their 20,000 limit.